reference/conanfile/attributes.rst

```diff
@@ -1809,6 +1809,32 @@ When ``True`` it enables the new run in a subsystem bash in Windows mechanism. :
         ...
         win_bash = True
 
+        def build(self):
+            self.run(cmd)  # will run <cmd> inside bash
+
+
+win_bash_run
+------------
+
+.. warning::
+
+    This is an **experimental** feature subject to breaking changes in future releases.
+
+Available since: `1.54.0 <https://github.com/conan-io/conan/releases/tag/1.54.0>`_
+
+When ``True`` it enables running commands in the ``"run"`` scope, to run them inside a bash shell.
```

Comment:
how do I enable it for an another scope?

Reply:
``win_bash`` is that is used for the other scope ("build")

Reply:
I think that's not completely clear, maybe we can add some clarification on that for `win_bash` and maybe adding here something like this is the equivalent of `win_bash` for the `run` scope ?